content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
```

Comment:
It's more of a style preference for me than anything. Most of the documentation I have read employs passive voice (AFAIK) and it just *feels* more serious/legit.

For example, I might write a tutorial in second person, "hey you there, I'm going to teach you a think, here's some things you can do, you now know these things"

Whereas "official documentation" feels more officially with passive voice to me